iroh-relay/src/server/client_conn.rs

```diff
@@ -66,9 +69,16 @@ impl ClientConn {
             stream: io,
             write_timeout,
             channel_capacity,
+            rate_limit: rate_limit_config,
             server_channel,
         } = config;
 
+        let quota = governor::Quota::per_second(rate_limit_config.bytes_per_second)
+            .allow_burst(rate_limit_config.max_burst_bytes);
+        // TODO: Allow creating this with mocked time for tests?
```

Comment:
you can use this: https://docs.rs/tokio/latest/tokio/time/fn.advance.html for testing tokio time

Reply:
Oh, I should have removed that TODO.

We can't because `governor::RateLimiter` has its own time source.  It does parametrise on that clock, which I started plugging in to get this tests better... only to discover they have a `ReasonablyRealtime` trait which must be implemented for the `until_n_ready` method we use and the fake clock source does not implement that.  I get that this is a bit difficult to do... so decided a test of 1.2s is the best we can do.  I'm hoping it isn't going to turn out to be flaky though.